content/kubermatic/main/tutorials-howtos/applications/add-remove-application-version/_index.en.md

```diff
@@ -0,0 +1,248 @@
++++
+title = "Add or Remove an Application Version"
+date =  2023-01-23T11:25:41+02:00
+weight = 3
++++
+
+This guide targets KKP Admins and details adding and removing a version to an `ApplicationDefinition`.
+
+## How to add a version to an ApplicationDefinition
+To make a new version of an application available, you only have to add it to `ApplicationDefinition` version's list.  
+Let's say you have the following `ApplicationDefinition`
+```yaml
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationDefinition
+metadata:
+  name: apache
+spec:
+  defaultValues:
+    commonLabels:
+      owner: somebody
+  description: Apache HTTP Server is an open-source HTTP server for modern operating systems
+  method: helm
+  versions:
+    - template:
+        source:
+          helm:
+            chartName: apache
+            chartVersion: 9.2.9
+            url: https://charts.bitnami.com/bitnami
+      version: 9.2.9
+```
+And want to make the new version `9.2.11` available. Then, all you have to do is to add the new version as described below:
+
+```yaml
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationDefinition
+metadata:
+  name: apache
+spec:
+  defaultValues:
+    commonLabels:
+      owner: somebody
+  description: Apache HTTP Server is an open-source HTTP server for modern operating systems
+  method: helm
+  versions:
+    - template:
+        source:
+          helm:
+            chartName: apache
+            chartVersion: 9.2.9
+            url: https://charts.bitnami.com/bitnami
+      version: 9.2.9
+    - template:
+        source:
+          helm:
+            chartName: apache
+            chartVersion: 9.2.11
+            url: https://charts.bitnami.com/bitnami
+      version: 9.2.11
+```
+Users will now be able to reference this version in their `ApplicationInstallation`. For additional details, see the [update an application guide]({{< ref "../update-application" >}}).
+
+{{% notice warning %}}
+Do not replace one version with another, as it will be perceived as a **deletion** by the application installation controller leading to **deletion of all `ApplicationInstallation` using this version.**
+For more details, see how to delete a version from an `ApplicationDefinition`.
+{{% /notice %}}
+
+## How to delete a version from an ApplicationDefinition
+Deleting a version from `ApplicationDefinition` will trigger the deletion of all `ApplicationInstallations` that reference this version! It guarantees that only desired versions are installed in user clusters, which is helpful if a version contains a critical security breach.
+Under normal circumstances, we recommend following the deprecation policy to delete a version.
+
+### Deprecation policy
+The deprecation policy is the following one:
+* stop the user from creating or upgrading to the deprecated version. But let them edit the application using a deprecated version. (it may be needed for operational purposes)
```

Comment:
```suggestion
* stop the user from creating or upgrading to the deprecated version. But let them edit the application using a deprecated version (it may be needed for operational purposes).
```